### DISCUSSION TITLE
Welcome to web3000_ungproject Discussions!

### DISCUSSION THREAD

**Discussion by tofutiger16** [tofutiger16/web3000_ungproject#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by demo1d** [comment]
https://gist.github.com/jo/8619441
Here are some encryption libraries for JavaScript that I have found we could look into.
